README.md

```diff
@@ -117,16 +117,19 @@ Some more important arguments are as follows:
 | global.kafka.install                    | Install Kafka and Zookeeper, **default=true**
 | global.redis.install                    | Install Redis, **default=true**
```

Comment:
Could global.redis.install &  global.redis-cluster.install be removed now?

Reply:
Those cannot be removed. The new switch is simply to select which should be used. The install flags still need to be there. It's annoying that the condition in Chart.yaml cannot support not (`!flag`). I scratched my head for a little bit about this one. I didn't want to play around with the Jenkinsfile yet.